delete.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#!/usr/bin/env bash
```

Comment:
why the double shebang?

Reply:
Thanks @thraxil for catching it. I've removed it!